web/src/routes/PoisPage.tsx

```diff
@@ -208,12 +221,12 @@ const PoisPage = ({ cityCode, languageCode, cityModel, pathname, languages }: Ci
       bboxViewport={moveViewToBBox(cityModel.boundingBox, defaultMercatorViewportConfig)}
       currentFeature={currentFeature}
       direction={direction}
+      overlayFooter={overlayFooter}
```

Comment:
Why is the footer passed as prop and not just defined in `MapsView`?

Reply:
because i would have to pass `cityCode` and `language` as additional props

Reply:
Would be the better solution imo as long as there are no plans to use the MapsView with a different footer. We only use react element props when we want the components to be used multiple similar ways.